modules/serverless-cost-management-labels.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * /serverless/integrations/serverless-cost-management-integration.adoc
+
+:_content-type: REFERENCE
+[id="serverless-cost-management-labels_{context}"]
+= Using labels for cost management queries
+
+Labels, which are referred to in cost management as _tags_, can be applied for nodes, namespaces or pods. Each label is a key and value pair. You can use a combination of multiple labels to generate reports. You can access reports about costs by using the link:https://console.redhat.com/openshift/cost-management/[Red Hat hybrid console].
+
+Labels are inherited by a node to namespace to pod hierarchy. However, labels are not overridden if they already exist on a resource. For example, Knative services have a default `app=<revision_name>` label:
```

Comment:
Maybe it's just a lack of knowledge in this area, but I don't understand what it means "inherited by a node to namespace to pod hierarchy." Not sure if there's a word missing or it needs reworded?

Reply:
reworded to say "Labels are inherited from nodes to namespaces, and from namespaces to pods." does that help?

Reply:
Much clearer, thanks!